### PR TITLE
Getting info about target branch for CircleCI presubmits

### PR DESCRIPTION
**Pull request by utka**

### CODE REVIEW

File: bin/ci2gubernator.sh [hunk 1]
```diff
@@ -70,7 +70,13 @@ fi
 
 if [ "${CIRCLE_BRANCH}" == "master" ] || [ "${CIRCLE_BRANCH}" == "release-1.2" ]; then
```
Comment (icygalz) [inline bin/ci2gubernator.sh]: This condition should only works for Postsubmit. We don't want to polute the presubmit results with some other presubmit tests we don't care.

Comment (utka) [reply bin/ci2gubernator.sh]: Done, added check for "${CI_PULL_REQUEST}" == "https://github.com/istio"*

**Comment by icygalz** [comment]
/lgtm

this is part of https://github.com/istio/istio/issues/13789

**Comment by howardjohn** [comment]
Not sure why just codecov failed, but it failed in the ci2gubernator part with 

```
go get: warning: modules disabled by GO111MODULE=auto in GOPATH/src;
	ignoring go.mod;
	see 'go help modules'
fatal: ref HEAD is not a symbolic ref
Exited with code 128
```

https://circleci.com/gh/istio/istio/441791?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link



The codecov task runs the tests, then checks out `HEAD^` and runs it again. So maybe that is why it breaks it

**Comment by howardjohn** [comment]
This is failing with 
```
bin/ci2gubernator.sh --exit_code=$(cat exit_code) --junit_xml=/go/out/tests/junit.xml
go get: warning: modules disabled by GO111MODULE=auto in GOPATH/src;
	ignoring go.mod;
	see 'go help modules'
fatal: ref HEAD is not a symbolic ref
Exited with code 128
```

and shouldn't be needed once we have https://github.com/istio/istio/pull/15068. Do we still need this or should we close?

**Pull request closed without merging**